model_compiler/python/nnir_to_openvx.py

```diff
@@ -2036,6 +2093,16 @@ def generateTestCPP(graph,argmaxOutput,fileName,virtual_tensor_flag):
                     if(!(err < maxError)) maxError = err;
                 }
             }
+            else if(data_type == VX_TYPE_INT64)
+            {
+                for(size_t i = 0; i < count; i++) {
+                    float src = (float)((long int*)ptr)[i];
```

Comment:
double check this statement. long int to float can result into undesired results